apps/backoffice-v2/src/lib/blocks/variants/OngoingBlocks/hooks/useOngoingBlocksLogic/useOngoingBlocksLogic.tsx

```diff
@@ -1,23 +1,23 @@
 import { useParams } from 'react-router-dom';
 import { useFilterId } from '@/common/hooks/useFilterId/useFilterId';
 import { useWorkflowByIdQuery } from '@/domains/workflows/hooks/queries/useWorkflowByIdQuery/useWorkflowByIdQuery';
-import { useCallback, useMemo } from 'react';
+import { useMemo } from 'react';
 import { useAmlBlock } from '@/lib/blocks/components/AmlBlock/hooks/useAmlBlock/useAmlBlock';
 import { createBlocksTyped } from '@/lib/blocks/create-blocks-typed/create-blocks-typed';
 
 export const useOngoingBlocksLogic = () => {
   const { entityId: workflowId } = useParams();
   const filterId = useFilterId();
+
   const { data: workflow, isLoading } = useWorkflowByIdQuery({
     workflowId: workflowId ?? '',
     filterId: filterId ?? '',
   });
-  const kycSessionKeys = Object.keys(workflow?.context?.pluginsOutput?.kyc_session ?? {});
-  const getAmlData = useCallback((key: string) => workflow?.context?.aml, [workflow?.context?.aml]);
-  const amlBlock = useAmlBlock({
-    sessionKeys: kycSessionKeys,
-    getAmlData,
-  });
+
+  const amlData = useMemo(() => workflow?.context?.aml, [workflow?.context?.aml]);
```

Comment:
Redundant `useMemo`. Would make a bit more sense if it included the array wrapping.

Reply:
You're right, I added the array into it.